site/playroom.config.js

```diff
@@ -23,9 +23,7 @@ module.exports = {
         ignoreOrder: true,
       }),
       new SkuWebpackPlugin({
-        include: ['lib', 'css', 'reset', 'themes'].map((dir) =>
-          path.join(__dirname, braidSrc, dir),
-        ),
+        include: [path.join(braidRoot, 'src')],
```

Comment:
Minor, but could make `braidRoot` => `braidSrc` and add the `src` folder to it. Looks like every usage is doing this manually